lib/kanta/migrations/postgresql/v03.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Kanta.Migrations.Postgresql.V03 do
+  @moduledoc """
+  Kanta V3 Migrations
+  """
+
+  use Ecto.Migration
+
+  @kanta_application_sources "kanta_application_sources"
+  @kanta_messages "kanta_messages"
+
+  def up(opts) do
+    Kanta.Migration.up(version: 2)
```

Comment:
Shouldn't that be `version: 3`?

Reply:
It should be `version: 2`, we don't want to run the same migration in a infinite loop